creator-node/src/config.js

```diff
@@ -731,6 +731,18 @@ const config = convict({
     format: Boolean,
     env: 'findCIDInNetworkEnabled',
     default: true
+  },
+  otelTracingEnabled: {
+    doc: 'enable OpenTelemetry tracing',
+    format: Boolean,
+    env: 'otelTracingEnabled',
+    default: true
+  },
+  otelCollectorUrl: {
+    doc: 'the url for the OpenTelemetry collector',
+    format: String,
+    env: 'otelCollectorUrl',
```

Comment:
so there'll be a separate `audius-docker-compose` repo PR to set this for stage and env right
what's it gonna be set to for local?

Reply:
Yup, @dmanjunath got DNS set up for the collector in both stage and prod so the next PR will be setting this configs to the DNS values he made. 

For local we're still deciding but I think it'll be the url to a local Jaeger instance. We can spin one up with the monitoring stack

Reply:
kk sg! thx for update